src/applications/vaos/actions/appointments.js

```diff
@@ -65,6 +94,14 @@ export function fetchFutureAppointments() {
           data,
           today: moment(),
         });
+
+        // Fetch request messages
+        const pendingAppointments = data[2];
+        if (pendingAppointments?.length) {
+          pendingAppointments.forEach(a =>
```

Comment:
Is it possible to trigger this when someone clicks the "Show more" button? That way we don't request a bunch of messages up front.